homework-12/ToDoList/main.js

```diff
@@ -0,0 +1,39 @@
+const addTask = document.querySelector("#addtask");
+const taskTitle = document.querySelector("#taskTitle");
+const uList = document.querySelector("#ulList");

```

Comment:
```suggestion
const taskList = document.querySelector("#ulList");
```
Or something similar